framework/db/pgsql/QueryBuilder.php

```diff
@@ -245,17 +245,21 @@ public function update($table, $columns, $condition, &$params)
     /**
      * Normalizes data to be saved into the table, performing extra preparations and type converting, if necessary.
      * @param string $table the table that data will be saved into.
-     * @param array $columns the column data (name => value) to be saved into the table.
+     * @param array|\yii\db\Query $columns the column data (name => value) to be saved into the table or instance
+     * of \yii\db\Query to perform INSERT INTO ... SELECT SQL statement.
+     * Passing of \yii\db\Query is available since version 2.0.11.
      * @return array normalized columns
      * @since 2.0.9
      */
     private function normalizeTableRowData($table, $columns)
     {
-        if (($tableSchema = $this->db->getSchema()->getTableSchema($table)) !== null) {
```

Comment:
no need to check `=== false`. Also better not wrap the whole function in the if but return early, so change this to:

```php
if ($columns instanceof \yii\db\Query) {
    return $columns;
}
```